games/xworld3d/maps/xworld3d_env.py

```diff
@@ -123,39 +123,32 @@ def set_entity(self, type, loc=None, name=None):
         """
```

Comment:
The class xworld3d_env seems to share many common functions/codes with xworld2d_env. It might be better to maintain if a base class can be extracted from them, which is easier to maintain.
We could leave this to be done in the future.